middleware/allRoutes/handler.go

```diff
@@ -40,14 +40,21 @@ func Handler(routesHandler map[string]http.Handler, zebedeeClient *client.Client
 				return
 			}
 
-			// Construct contentPath with any colletion if present in cookie
+			if strings.HasSuffix(req.URL.String(), "/latest") {
```

Comment:
I would also move this block above the previous two blocks (no point in doing those regex's for these)

Reply:
Good point - Done 👍